installers/olm/config/redhat/related-images.yaml

```diff
@@ -0,0 +1,23 @@
+# https://github.com/operator-framework/operator-lifecycle-manager/blob/0.13.0/doc/contributors/design-proposals/related-images.md
+# https://redhat-connect.gitbook.io/certified-operator-guide/troubleshooting-and-resources/offline-enabled-operators
```

Comment:
📝 This is a Kustomize patch in the Red Hat bundle, but it might be part of the main installer (depending on https://github.com/CrunchyData/postgres-operator/pull/2558#discussion_r674384387).

Reply:
The Red Hat bundle includes more related images than the community version and points at their registry.